src/methods/zonal.jl

```diff
@@ -108,16 +108,31 @@ function _zonal(f, x::RasterStackOrArray, ::Nothing, data; progress=true, thread
     geoms = _get_geometries(data, geometrycolumn)
     n = length(geoms)
     n == 0 && return []
-    zs = _alloc_zonal(f, x, first(geoms), n; kw...)
-    _run(1:n, threaded, progress, "Applying $f to each geometry...") do i
+    zs = _alloc_zonal(f, x, geoms, n; kw...)
+    start_index = findfirst(x -> !ismissing(x), zs)
```

Comment:
Can we just return `start_index` from `_alloc_zonal` (e.g. `nmissing + 1`) to avoid the `findfirst` ?

Reply:
That makes more sense, I've added that to the most recent commit.